addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -452,14 +452,18 @@ body.editor_enable.editor_has_snippets {
             }
             .oe_snippet_thumbnail_img {
                 flex-shrink: 0;
-                width: 40px;
-                height: 32px;
+                width: 41px;
+                height: 30px; // 82x60 -> 41x30
                 padding: 0;
             }
-            .o_delete_btn {
+            .o_rename_btn, .o_delete_btn {
                 @extend %we-generic-link;
                 padding-top: 0;
                 padding-bottom: 0;
+                padding-left: 0;
+            }
+            &:not(:hover) .o_rename_btn, &:not(:hover) .o_delete_btn {
+                display: none;
```

Comment:
If needed like that, it should be written

```scss
&:not(:hover) {
    .o_rename_btn, .o_delete_btn {
```

but is not there a common selector that would match the rename and delete button? If not, you can a a common class.

Reply:
Indeed, there is an existing .btn class and I believe this "hidden until hovered" will need to apply to any button in there